docs/topics/Gateway.md

```diff
@@ -679,7 +680,7 @@ The inner `d` key is a boolean that indicates whether the session may be resumab
 
 #### Channel Create
 
-Sent when a new channel is created, relevant to the current user. The inner payload is a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object.
+Sent when a new channel is created, relevant to the current user. The inner payload is a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object. Does not fire for DMs.
```

Comment:
Would it be simpler to say that this only fires for guild channels, unless there is another hidden channel type we don't know about?

Reply:
maybe
```suggestion
Sent when a new guild channel is created, relevant to the current user. The inner payload is a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object.
```